lib/lattice.py

```diff
@@ -255,7 +281,8 @@ class DiamondLattice:
                     (2, 5), (3, 6), (4, 7), (5, 0),
                     (5, 3), (5, 4), (6, 0), (6, 2),
                     (6, 4), (7, 0), (7, 2), (7, 3)}
-
+    mpc = None
+    box_l = None
```

Comment:
I would advise against this change: `mpc` and `box_l` should be properties of the instantiated objects, not of the class.